app/src/main/java/edu/usf/sas/pal/muser/util/EventUtils.kt

```diff
@@ -35,11 +35,13 @@ object EventUtils {
      * @return UiAction object
      */
     @JvmStatic
-    fun newUiEvent(song: Song, capturedUiAction: UiEventType, context: Context): UiEvent {
+    fun newUiEvent(song: Song, capturedUiAction: UiEventType, context: Context, position: Long = 0): UiEvent {
```

Comment:
I would use a value other than `0` here for the "null" value because `0` is a valid value - probably best just to make it nullable, or use Long.MAX_VALUE.

Reply:
Let's also add another method here that has the original signature too, and that way code scattered throughout the app can keep the original method signature and not have to pass in "null" values - that would be handled in one place in this method. So you'd have another method like:

```kotlin
fun newUiEvent(song: Song, capturedUiAction: UiEventType, context: Context): UiEvent {
       newUiEvent(song, capturedUiAction, context, Long.MAX_VALUE)
}